breadcord/helpers.py

```diff
@@ -12,7 +11,10 @@
 
 if TYPE_CHECKING:
     # noinspection PyProtectedMember
-    from discord.ui.button import V, Button
+    from collections.abc import Sequence
```

Comment:
Minor issue with the noinspection comment getting displaced here, but should be a simple fix